common/religion/religions/arcane_group.txt

```diff
@@ -255,17 +258,40 @@
 			holy_site = sunstrider_isle
 			holy_site = andaroth
 			holy_site = dalaran
-			
-			#Main Group
-			doctrine = doctrine_spiritual_head
-			doctrine = doctrine_theocracy_temporal
 
 			doctrine = tenet_sun_worship
 			doctrine = tenet_eastern_kingdoms_syncretism
 			doctrine = tenet_astrology
 
+			#Main Group
+			doctrine = doctrine_spiritual_head
+			doctrine = doctrine_theocracy_temporal
+
 			#Magic
 			doctrine = doctrine_light_magic_accepted
+
+			#Clerical Functions
+			doctrine = doctrine_clerical_succession_temporal_fixed_appointment
+		}
+
+		cult_of_nightwell = {
+			#Main Group
+			color = rgb { 70 54 248 }
+			#religious_head = d_cult_of_nighwell
+			#icon = wcmw_nightwell_religion
```

Comment:
```suggestion
			icon = wcmw_nightwell_religion
```

Reply:
Thank you, you are fantastic. Let's get married.